library/system.lisp

```diff
@@ -76,55 +76,56 @@ While the result will always contain microseconds, some implementations may retu
 
   (declare get-run-time (Unit -> UFix))
   (define (get-run-time)
-    "Gets the run-time."
+    "Gets the run-time in microseconds."
     (lisp UFix ()
       (cl:get-internal-run-time)))
 
   (declare get-real-time (Unit -> UFix))
```

Comment:
this is the equivalent of `monotonic-bytes-consed` but for time. maybe parity with that API?

problem here is that "real time" or "run time" don't really have meaning unless one is intimately familiar with the Lisp standard.